.github/wiki/CodeMagic.md

```diff
@@ -0,0 +1,79 @@
+# CodeMagic
+
+Use the CodeMagic template to start a new project with CodeMagic as the CI/CD tool.
+
+## Workflows and Steps
+
+Out of the box, the CodeMagic Template has the following workflows and steps:
+
+| test                      | deploy_app_store                                        | deploy_staging                          | deploy_release_firebase                   |
+| ------------------------- | ------------------------------------------------------- | --------------------------------------- | ----------------------------------------- |
+| Install bundle            | Install bundle                                          | Install bundle                          | Install bundle                            |
+| Run CocoaPods install     | Run CocoaPods install                                   | Run CocoaPods install                   | Run CocoaPods install                     |
+| Fastlane - Build and Test | Fastlane - Build and Test                               | Fastlane - Build and Test               | Fastlane - Build and Test                 |
+| Fastlane - Clean Up Xcov  | Fastlane Match                                          | Fastlane Match                          | Fastlane Match                            |
+| Danger                    | Fastlane - Build and Upload Production App to App Store | Fastlane - Build and Upload Staging App | Fastlane: Build and Upload Production App |
```

Comment:
for `deploy_staging` type should we menntion some name where the generated files will be uploaded like other columns? Also we can update the `deploy_release_firebase` columns last row like below.
```suggestion
| Danger                    | Fastlane - Build and Upload Production App to App Store | Fastlane - Build and Upload Staging App | Fastlane: Build and Upload Production App to Firebase|
```